### PR TITLE
Fixes #7580: Replaced break-all with break-word

### LINKED ISSUE #7580: Incorrect text wrapping on an add-on details card in Disco Pane
Prerequisites:
extensions.webservice.discoverURL pref points to the -dev server in about:config

STR:

1. Open your Firefox browser
2. Navigate to about:addons
3. Observe the description for any of the listed add-ons - i.e Awesome Screenshot Plus

Actual result:
Text for the description and custom heading are not correctly wrapped

Expected result:
There are no text layout issues in the add-on cards

Notes:
- reproduced on AMO -dev with FF65, WIn10x64

image

### PR DESCRIPTION
**Pull request by suhailsinghbains**
Fixes #7580 
Replaced break-all with break-word in src/disco/components/Addon/styles.scss file at L23

### CODE REVIEW

**Comment by suhailsinghbains** [comment]
After the change
photo_2019-02-13_10-14-59

File: src/disco/components/Addon/styles.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@ $addon-padding: 20px;
     color: $header-copy-font-color;
     font-size: 12px;
     margin: 0.75em 0;
-    word-break: break-all;
+    word-break: break-word;
```
Comment (suhailsinghbains) [inline src/disco/components/Addon/styles.scss]: @willdurand Can we use word-wrap property to solve [0]
0:https://github.com/mozilla/addons-frontend/issues/7580#issuecomment-464088331

Comment (willdurand) [reply src/disco/components/Addon/styles.scss]: ohhh, I guess yes. Wanna try it?

Comment (suhailsinghbains) [reply src/disco/components/Addon/styles.scss]: Yes, Sure :)

**Pull request merged**